subprojects/gst-plugins-good/sys/v4l2/gstv4l2object.c

```diff
@@ -5214,7 +5214,11 @@ gst_v4l2_object_decide_allocation (GstV4l2Object * obj, GstQuery * query)
       gst_v4l2_buffer_pool_copy_at_threshold (GST_V4L2_BUFFER_POOL (pool),
           FALSE);
     }
-
+  } else if (V4L2_TYPE_IS_CAPTURE(obj->type) && (obj->mode == GST_V4L2_IO_DMABUF_IMPORT)) {
+    // since we reuse right index, so let's keep buffer number of downstream pool same with v4l2buffer pool
+    GST_DEBUG ("got min: %d own_min: %d max: %d min_buffers:%d", min, own_min, max, obj->min_buffers);
```

Comment:
Again use GST_DEBUG_OBJECT(), use C99 comments `/* */` to match this file style.